registration/forms.py

```diff
@@ -412,6 +412,11 @@ def update_user(self, application, commit=True):
                 application.save()
 
         if self.restart_fields_changed():
+            if not application.email_addresses_verified():
+                messages.add_message(self.request, messages.INFO,
+                                     "Thank you for submitting these changes - we will build your instance to "
+                                     "apply them once your email address is confirmed.")
```

Comment:
@viadanna, nit: suggestion.

```suggestion
                                     "apply them once your email address is verified.")
```